test/built-ins/TypedArray/prototype/copyWithin/resizable-buffer.js

```diff
@@ -0,0 +1,183 @@
+// Copyright 2023 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-%typedarray%.prototype.copywithin
+description: >
+  TypedArray.p.copyWithin behaves correctly when the receiver is backed by
+  resizable buffer
+includes: [compareArray.js, resizableArrayBufferUtils.js]
+features: [resizable-arraybuffer]
+---*/
+
+const TypedArrayCopyWithinHelper = (ta, ...rest) => {
+  ta.copyWithin(...rest);
+};
```

Comment:
If we were writing these tests from scratch, I'd suggest getting rid of this function — unlike the ArrayCopyWithinHelper above it doesn't really make the test easier to read or write. But I suspect that's probably on the wrong side of the 'too picky' line.

Reply:
Done:
Removed the unnecessary abstraction.